eng/ignore-links.txt

```diff
@@ -7,3 +7,16 @@ https://docs.microsoft.com/javascript/api/@azure/storage-blob-changefeed
 https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/storage/storage-blob-changefeed/samples/javascript
 https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/storage/storage-blob-changefeed/samples/typescript
 https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/storage/storage-blob-changefeed/test/
+https://azuresdkdocs.blob.core.windows.net/$web/javascript/azure-service-bus/7.0.0-preview.5/classes/servicebusclient.html
```

Comment:
In general we should avoid using the ignore-links in this way. We should not be linking to something that does not yet exist. 

As for these particular links we should actually avoid linking to these as these readmes will end up on docs.microsoft.com and we don't want those going back into our github.io generated docs. I suggest we use a general docs link and remove these specific api links in our readmes.

Reply:
> As for these particular links we should actually avoid linking to these as these readmes will end up on docs.microsoft.com

For previews too?

Reply:
@scbedd please correct me if I'm wrong but I do believe we are also publishing all the readmes for preview releases to docs.microsoft.com.

Reply:
This problem of non-existing links could be present even for the non-preview releases.
Say, a new class is added to the API, and we refer that class in the readme with the future link of docs.microsoft.com.
Before publishing the package, the new link won't exist.

Should we avoid adding references to the future `docs.microsoft.com` links too in the readme then?

Reply:
Yes in general we should avoid adding references to anything that doesn't yet exist. We should provide a generic link to the docs landing page that will always exist. Later if we feel it is critical then we can add links to particular api references docs but in most cases I don't expect that to be very interesting in readmes and can be pretty fragile overtime.

Reply:
Thanks, @weshaggard, makes sense. I've logged an issue for service-bus to avoid adding links, to keep it concise.
https://github.com/Azure/azure-sdk-for-js/issues/10551